h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -171,6 +173,32 @@ public StackedEnsembleParametersV99 fillFromImpl(StackedEnsembleModel.StackedEns
             paramsSchema = new DeepLearningV3.DeepLearningParametersV3();
             params = new DeepLearningModel.DeepLearningParameters();
             break;
+          case naivebayes:
+            paramsSchema = new NaiveBayesV3.NaiveBayesParametersV3();
+            params = new NaiveBayesModel.NaiveBayesParameters();
+            break;
+          case xgboost:
```

Comment:
@sebhrusen Not sure how to solve this: XGBoost is an extension so I think we should import it dynamically but the solution isn't the prettiest. Another solution that occurred to me is to (ab)use the `hex.ensemble.Metalearners.LocalProvider` and add two factory method to return `ModelParametersSchemaV3` and `Model.Parameters`.

Or do you have better ideas how to solve this issue?

Reply:
yeah, my fault, I forgot the parameters schema when creating this provider.
I still prefer to have a single entry point to load this dynamically, and `Metalearners` makes sense here.

for `params`, you can already use `Metalearners.createParameters()`:you just need to make it public.
for the schema, I think it's ok to extend the `LocalProvider` with a method instantiating schemas, without being a complete abuse :) Another solution would be to use reflection as all model schemas follow the same convention: `hex.schemas.MyModelV3.MyModelParametersV3` which will work, except potentially for users that would want to plug their own algo...